packages/marketplace-ui/src/hooks/useBountyInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import { PublicKey } from "@solana/web3.js";
+import { MarketplaceSdk } from "@strata-foundation/marketplace-sdk";
+import {
+  useMint, useReserveAmount, useTokenBondingFromMint,
+  useTokenMetadata
+} from "@strata-foundation/react";
+import { toNumber } from "@strata-foundation/spl-token-bonding";
+import { SplTokenMetadata } from "@strata-foundation/spl-utils";
+import debounce from "lodash/debounce";
+import React, { useMemo } from "react";
+
+export function useBountyInfo(mintKey: PublicKey | undefined): {
+  info: {
+    name: string | undefined;
+    image: string |undefined;
+    createdAt: Date | undefined;
+    description: string | undefined;
+    fundsHaveBeenUsed: boolean | undefined;
+    bountyClosed: boolean | undefined;
+    contributed: number | undefined;
+    attributes: Record<string, string | number> | undefined;
+    isNormalBounty: boolean;
+  };
+  loading: boolean;
+} {
+  const { info: tokenBonding, loading: bondingLoading } =
+    useTokenBondingFromMint(mintKey);
+  const {
+    data: targetData,
+    metadata,
+    loading: targetMetaLoading,
+    image,
+    displayName
+  } = useTokenMetadata(mintKey);
+  const reserveAmount = useReserveAmount(tokenBonding?.publicKey);
+  const baseMint = useMint(tokenBonding?.baseMint);
+  // Debounce because this can cause it to flash a notification when reserves change at the
+  // same time as bonding, but one comes through before the other.
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const fundsHaveBeenUsed = useMemo(
+    debounce(
+      () =>
+        tokenBonding &&
+        baseMint &&
+        reserveAmount &&
+        toNumber(tokenBonding.reserveBalanceFromBonding, baseMint) !==
+          reserveAmount,
+      500
+    ),
+    [tokenBonding, baseMint, reserveAmount]
+  );
```

Comment:
Does this debounce work? Maybe we want to pull duplicate this in our hooks directory. https://github.com/SevenOutman/use-debounced-memo/blob/master/src/useDebouncedMemo.ts